config/routes.rb

```diff
@@ -25,6 +25,7 @@
     end
 
     resource :user, only: [:show]
+    resource :id_card_attributes, only: [:show]
```

Comment:
Would it be better to call it `id_card` and not `id_card_attributes`? attributes sounds like it would belong to a restful resource, not be the restful resource

Reply:
Maybe? In terms of the whole ecosystem, we're not letting the user request and ID card, we're only providing the verified attributes needed to initiate the ID card request, so this feels more accurate to me. And kind of hedging against a possible future where more of the ID card functionality lives in vets.gov proper.

Reply:
If you do envision having an id card in the future it might look something like:

veteran_card/:some_id:/attributes

where attributes would be a nested resource or more likely, be part of the id card object as part of a complex object.